frontend/src/forms/ComponentLike/preInjectedFiles.tsx

```diff
@@ -1,11 +1,10 @@
-import { Box, Button, Icon } from "@material-ui/core";
+import { Box, Button, Icon, Typography } from "@material-ui/core";
```

Comment:
@GongDDEX 
Please start with `Label.tsx`. If the labels do not work for some scenarios, please let me know.